### ISSUE TITLE
After  user applies a filter on Help Board, the user focus should be redirected to the  filtered post

### ISSUE THREAD

**Issue by sheeba1** [FightPandemics/FightPandemics#1256]
Steps:
1. Navigate to https://rcv11.fightpandemics.xyz 
 2.Navigate to View Help Board
3. Scroll Down a few posts on Help Board
4. Click on Filters-> location
5. Enter 'Irvine' ->choose Irvine, CA 92602
6.Click View Result

Filtered Post goes to the top. But the user observes some scroll on the page but not redirected to the filtered post. (I can share video on slack)

Expected: User should be redirected to the filtered posts

**Comment by joshmorel** [comment]
@sheeba1 I'm not seeing the issue here. Can you share a video?

**Comment by OzkanAbdullahoglu** [comment]
reproduced like in below gif.

![2020-07-23_01-29-23](https://user-images.githubusercontent.com/34690158/88236233-329ba800-cc85-11ea-91c2-7f1ff0d3ea93.gif)

**Comment by sheeba1** [comment]
@OzkanAbdullahoglu , thanks for reproducing it and adding to Git.(reproduced exactly  !).  Ididnt know how to add a video

**Comment by joshmorel** [comment]
@sheeba1 & @OzkanAbdullahoglu I can confirm this works with Firefox but not Chrome.

I was wondering if it is because the antd layout content is using non-standard ::--webkit-scrollbar pseudoelement? It works fine in Firefox using the main scrollbar so maybe if there's a way to disable it so it behaves similar in other browsers? I couldn't immediately see anything here: https://ant.design/components/layout/


OR if that doesn't work maybe just scroll to top when filters applied? https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo

**Comment by OzkanAbdullahoglu** [comment]
> @sheeba1 & @OzkanAbdullahoglu I can confirm this works with Firefox but not Chrome.
> 
> I was wondering if it is because the antd layout content is using non-standard ::--webkit-scrollbar pseudoelement? It works fine in Firefox using the main scrollbar so maybe if there's a way to disable it so it behaves similar in other browsers? I couldn't immediately see anything here: https://ant.design/components/layout/
> 
> OR if that doesn't work maybe just scroll to top when filters applied? https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo

stroll to the top can be a solution but I 'll dig a bit when I find the time, the first solution which you mentioned would be better I think.

**Comment by joshmorel** [comment]
Related - but #1372 also affects Firefox.
